exasol/secret_store.py

```diff
@@ -28,12 +28,16 @@ def close(self) -> None:
             self._con.close()
             self._con = None
 
-    def connection(self) -> sqlcipher.Connection:
+    def connection(
+        self,
+    ) -> sqlcipher.Connection:  # pylint: disable=E1101
```

Comment:
Add a short explanation & justification for disabling pylint here

Reply:
We were struggling with adding this explanation, but formatter inserted line breaks and moved disblable instruction then did not disable the linter anymore. :frowning: 
I add comment above. :slightly_smiling_face: